File: Cryptography and security libraries and algorithms/C/Code complete/PRNG.c
#include <stdlib.h>
#include <openssl/rand.h>
#include <sys/random.h>
//Generate a winning lottery number between 0 and 1000
int generateWinningNumber() {
    int winningNumber;
    //Generate a random number between 0 and 1000 using the OpenSSL library.
    RAND_bytes((unsigned char *) &winningNumber, sizeof(winningNumber));

    //Add 1 to the generated number since it's 0-indexed but we want 1-indexed.
    winningNumber++;

    return winningNumber % 1001;  //returns a value between 1 and 1000 (inclusive)
}

File: Cryptography and security libraries and algorithms/Python/Code complete/PRNG.py
import secrets
from random import randrange
#Generate a winning lottery number between 0 and 10000
winning_number = randrange(0, 10000)
#Ask the user to guess a number between 0 and 100
guess_count = 1
while True:
    try:  #check if the input is an integer or not. If it's not, raise an exception. Otherwise continue with code below.
        guess = int(input('Guess a number between 0 and 10000
